### PR TITLE
Continuating french translate

### PR DESCRIPTION
**Pull request by mariogamer2**

### CODE REVIEW

File: lang/fr.json [hunk 1]
```diff
@@ -33,15 +33,222 @@
...
+"Inject unencrypted TWL (DSi) FAT12 photo partition (TWL logical partition 1) from file.":"",
+"Inject unencrypted AGB (GBA) savegame partition from file.":"",
+"Inject unencrypted first firmware partition from file.":"",
+"Inject unencrypted second firmware partition from file.":"",
+"Inject unencrypted CTR partition from file.":"",
+"Inject unencrypted CTR (3DS) FAT 16 partition (CTR logical partition 0) from file.":"",
+"Inject files to the CTRNAND FAT16 partition.":"",
+"Copy":"",
+"Copy options.":"",
+"SysNAND to EmuNAND":"",
```
Comment (dukesrg) [inline lang/fr.json]: You should not keep like that, there will be an empty string as a result when French choosen

Comment (mariogamer2) [inline lang/fr.json]: I know, I just do this to be able to finish it latter.

**Comment by mariogamer2** [comment]
Must be finished today (not shure)

File: lang/fr.json [hunk 2]
```diff
@@ -67,25 +67,25 @@
...
+"Dump NAND image, separate partitions or files from the third EmuNAND.":"Extraire l'image de la NAND, des partitions séparées ou des fichiers à partir de la troisième EmuNAND.",
+"Image":"Image",
 "Backup NAND raw image to file.":"",
-"Partitions":"",
-"Dump decrypted NAND partitions to file.":"",
-"Dump decrypted TWL (DSi) partition to file.":"",
-"Dump decrypted TWL (DSi) FAT16 partition (TWL logical partition 0) to file.":"",
-"Dump decrypted TWL (DSi) FAT12 photo partition (TWL logical partition 1) to file.":"",
-"Dump decrypted AGB (GBA) savegame partition to file.":"",
+"Partitions":"Partitions",
```
Comment (dukesrg) [inline lang/fr.json]: If string is exactly the same as in english variant, it should not be placed inside of the translation.

**Comment by mariogamer2** [comment]
Ok now will contact motezazer...

**Comment by mariogamer2** [comment]
@dukesrg Everything must be  ok now.

**Comment by mariogamer2** [comment]
I done the merge cleanly. This is good. I thank some people for some fixes.

File: lang/fr.json [hunk 3]
```diff
@@ -237,10 +237,10 @@
 "FAILED": "Échec!\n",
 "COMPLETED": "Complété!\n",
 "SETTINGS": "Options",
-"FORCE_UI_BOOT": "Forcer le démarrage de l'interphace graphique (GUI)    %ls",
+"FORCE_UI_BOOT": "Forcer le démarrage de l'interface graphique (GUI)    %ls",
```
Comment (dukesrg) [inline lang/fr.json]: CAPS keys is from old translation - kept for compatibility until old console elimunated.

File: lang/fr.json [hunk 4]
```diff
@@ -140,7 +140,7 @@
 "Settings":"Paramètres",
 "Change settings.":"Modifier les paramètres.",
 "Default boot":"Démarrage par défaut",
-"Force rxTools UI":"Forcer le démarrage de l'interphace graphique (GUI)",
+"Force rxTools UI":"Forcer le démarrage de l'interface graphique (GUI)",
```
Comment (dukesrg) [inline lang/fr.json]: Such long will take 2 or more strings in settings menu page, please keep as short as possible.

File: lang/fr.json (hunk 3 above)
Comment (mariogamer2) [inline lang/fr.json]: ?

**Comment by mariogamer2** [comment]
pushed

**Comment by mariogamer2** [comment]
Good

**Comment by mariogamer2** [comment]
@dukesrg ?

**Comment by mariogamer2** [comment]
Does there is anything who is not good ?

**Comment by mariogamer2** [comment]
Does it must be merged in dev, or anything else?

**Comment by dukesrg** [comment]
If it's ready I could merge it to the master, why not. Only doubt about long menu captions which could force two line menu options

**Comment by mariogamer2** [comment]
Where? Yes, it is ready.

**Comment by mariogamer2** [comment]
Column number are like ru.json,so I thinf it can work.

**Pull request merged**